File: Raven.Client.Java/src/main/java/net/ravendb/abstractions/data/UuidType.java
package net.ravendb.abstractions.data;

import net.ravendb.abstractions.basic.UseSharpEnum;

@UseSharpEnum
public enum UuidType {

  DOCUMENTS((byte)1),
  ATTACHMENTS((byte)2),
  DOCUMENTTRANSACTIONS((byte)3),
  MAPPEDRESULTS((byte)4),
  REDUCERESULTS((byte)5),
  SCHEDULEDREDUCTIONS((byte)6),
  QUEUE((byte)7),
  TASKS((byte)8),
  INDEXING((byte)9),
  ETAGSYNCHRONIZATION((byte)10);

  private byte value;

  private UuidType(byte value) {
    this.value = value;
  }

  /**
   * @return the value
   */
  public byte getValue() {
    return value;
  }


}

File: Raven.Abstractions/Data/UuidType.cs
namespace Raven.Abstractions.Data
{
    public enum UuidType : byte
    {
        Documents = 1,
        Attachments = 2,
        DocumentTransactions = 3,
        MappedResults = 4,
        ReduceResults = 5,
        ScheduledReductions = 6,
        Queue = 7,
        Tasks = 8,
        Indexing = 9,
		EtagSynchronization = 10,
		DocumentReferences = 11
    }
}
